up42/workflow.py

```diff
@@ -229,7 +235,45 @@ def add_workflow_tasks(self, input_tasks: Union[List[str], List[Dict]]) -> None:
             f"{self.workflow_id}/tasks/"
         )
         self.auth._request(request_type="POST", url=url, data=input_tasks)
-        logger.info("Added tasks to workflow: %r", input_tasks)
+        logger.info(f"Added tasks to workflow: {input_tasks}")
+
+        if return_parameters:
+            default_parameters = self._get_default_parameters()
+            logger.info(f"Returning default workflow parameters: {default_parameters}")
+            return default_parameters
+
+    def add_template(self, template_name: str, return_parameters: bool = True):
+        """
+        Adds workflow template tasks to workflow.
+
+        Args:
+            template: The template name. Get via up42.get_templates().
+
+        Returns:
+            Optional: Default workflow parameters.
+        """
+        # TODO: Rename add_workflow_template?
+        # TODO: Blocks not available on staging!
+        templates = self.get_templates(basic=True)
+        if not template_name in list(templates.keys()):
+            raise ValueError(
+                "The selected template is not available, please use up42.get_templates()."
+            )
+
+        input_tasks = templates[template_name]
+        logging.getLogger("up42.workflow").setLevel(logging.CRITICAL)
+        self.add_workflow_tasks(input_tasks=input_tasks, return_parameters=False)
+        logging.getLogger("up42.workflow").setLevel(logging.INFO)
+        logger.info(
+            f"Add template {template_name} to workflow containing tasks: {input_tasks}"
+        )
+
+        if return_parameters:
+            template_parameters = self.get_templates(basic=False)[template_name]
+            logger.info(
+                f"Returning template workflow parameters of template {template_name}: {template_parameters}"
+            )
+            return template_parameters
```

Comment:
Same here. These are methods of the workflow class, if they are adding_template to workflow, this should be the only thing they do as I see it.